api/v1alpha1/release_types.go

```diff
@@ -44,6 +45,10 @@ const (
 	// releaseConditionType is the type used when setting a release status condition
 	releaseConditionType string = "Succeeded"
 
+	// BindingDeploymentStatusConditionType is the condition type to retrieve from the ComponentDeploymentConditions
+	// in the SnapshotEnvironmentBinding's status to copy into the Release status
+	BindingDeploymentStatusConditionType string = "AllComponentsDeployed"
```

Comment:
The type should not be exported. Same as the release type.

Reply:
Needs to be. It has to be accessible in `gitops/utils.go`

Reply:
Fair enough. Missed that.